src/OpenColorIO/ops/Lut3D/Lut3DOpData.h

```diff
@@ -66,8 +66,8 @@ class Lut3DOpData : public OpData
 
     // Use functional composition to generate a single op that 
     // approximates the effect of the pair of ops.
-    static Lut3DOpDataRcPtr Compose(ConstLut3DOpDataRcPtr & A,
-                                    ConstLut3DOpDataRcPtr & B);
+    static void Compose(Lut3DOpDataRcPtr & A,
```

Comment:
It seems better to keep LUT code in sync and thus use the same PR.

Reply:
My suggestion at moving these is because this PR is targeting Lut1D refactoring, but there are also several Lut3D changes present. #642 on the other hand targets Lut3D. I understand your point about keeping API's in sync, and it isn't unreasonable. My hesitation was more that this involves squashing Lut3D commits into a Lut1D merge, which makes history a bit less clear.

Reply:
Michael, it's a good suggestion, we will look at doing this and see if there are any issues.

Reply:
Removed from this PR